backend/src/main/java/com/redhat/cloud/notifications/recipients/rbac/AuthRequestFilter.java

```diff
@@ -11,14 +12,13 @@
 class AuthRequestFilter implements ClientRequestFilter {
 
     String authInfo;
-
-    @ConfigProperty(name = "rbac.service-to-service.secret", defaultValue = "addme")
     String secret;
-
-    @ConfigProperty(name = "rbac.service-to-service.application", defaultValue = "notifications")
     String application;
 
     AuthRequestFilter() {
+        Config config = ConfigProvider.getConfig();
+        secret = config.getOptionalValue("rbac.service-to-service.secret", String.class).orElse("addme");
+        application = config.getOptionalValue("rbac.service-to-service.application", String.class).orElse("notifications");
```

Comment:
The `@ConfigProperty` wasn't working, I was getting `null` on these two variables (same issue in CI). Probably related to the way this class is used (`@RegisterProvider(AuthRequestFilter.class)`)